test/winter_onboarding_2021/fleet_management_service/db/fleet_test.clj

```diff
@@ -17,7 +40,39 @@
           user-id (:users/id (user-db/create user))
           fleet {:name "Azkaban Fleet 1"
                  :created-by user-id}]
-      (fleet-db/create fleet)
       (is (= #:fleets{:name "Azkaban Fleet 1"
-                      :created-by user-id} (select-keys (first (db-core/query! ["select * from fleets;"]))
-                                                        [:fleets/created-by :fleets/name]))))))
+                      :created-by user-id} (select-keys (fleet-db/create fleet)
+                                                        [:fleets/name :fleets/created-by]))))))
+
+(deftest select-fleets-for-admin
+  (testing "Should fetch us a list of first 10 fleets related to an admin user"
```

Comment:
I think, the testing block sentence should be improved indicating 10 refers to page size may be something like `Should return list of fleets in first page which are related to admin user` . wdyt?

Reply:
So this test is not testing any pagination logic. It just asks for 10 fleets ordered by `created-at` & tests on them